nss-day-cohort-44/nutshell-whiskered-gnomes
# Ks chat form

# Description
- adds message form upon Nutshell page load
- adds functionality to listen for message form send button click
- adds functionality to save new message
- adds functionality to re-render messages to DOM

Fixes #7 

## Type of change

# Testing Instructions
git add and commit changes on your working branch
git checkout master
git fetch --all
git checkout ks-chat-form
serve
What you should see: Any messages stored in your api should render. You should be able to add a message to the input field, click send, and see the message render. Messages should render in order with newest at bottom of list.

# Checklist:
